src/content/operations/upgrading/_index.en.md

```diff
@@ -0,0 +1,57 @@
+---
+title: "Upgrading"
+date: 2023-09-06T14:06:11+02:00
+weight: 15
+---
+
+Starting with Submariner 0.16, the recommended way to upgrade Submariner is via the [`subctl upgrade`](#automated-upgrade) command.
+This can be used to upgrade clusters to Submariner 0.16.
```

Comment:
Maybe we can remove this sentence as this will become stale in the next release.

```suggestion
```